tests/app/views/contexts/summary/test_question.py

```diff
@@ -969,3 +1047,58 @@ def test_dynamic_answer_options(self):
 
                 # Then
                 self.assertEqual(question.answers[0]["value"], expected_output)
+
+    def test_get_answer_method(self):
+        data_set = [
+            # answer_schema, answer_store, expected_output
+            (
+                {
+                    "id": "building",
+                    "label": "Building",
+                    "type": "TextField",
+                },
+                AnswerStore(
+                    [{"answer_id": "building", "value": "Government Buildings"}]
+                ),
+                "Government Buildings",
+            ),
+            (
+                {
+                    "id": "building",
+                    "label": "Building",
+                    "type": "TextField",
+                    "default": "Government Buildings",
+                },
+                AnswerStore([]),
+                "Government Buildings",
+            ),
+        ]
+
+        for answer_schema, answer_store, expected_output in data_set:
+            schema = self.address_questionnaire_schema("Newline")
+            with self.subTest(
+                schema=schema,
+                answer_schema=answer_schema,
+                answer_store=answer_store,
+                expected_output=expected_output,
+            ):
+                # Given
+                question_schema = self.get_question_schema(answer_schema)
+
+                # When
+                question = Question(
+                    question_schema,
+                    answer_store=answer_store,
+                    schema=schema,
+                    rule_evaluator=self.get_rule_evaluator(),
+                    value_source_resolver=self.get_value_source_resolver(),
+                    location=None,
+                    block_id="address-group",
+                    return_to=None,
+                )
+
+                # Then
+                self.assertEqual(
+                    Question.get_answer(question, answer_store, "building"),
```

Comment:
How come this is not using the instance from above?.

Reply:
Now it is using that instance, thanks!